core/src/test/java/org/mskcc/cbio/oncokb/util/MutationEffectParameterizedTest.java

```diff
@@ -0,0 +1,93 @@
+package org.mskcc.cbio.oncokb.util;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.mskcc.cbio.oncokb.model.IndicatorQueryResp;
+import org.mskcc.cbio.oncokb.model.Query;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class MutationEffectParameterizedTest {
+    private static String VARIANT_SUMMARY_EXAMPLES_PATH = "src/test/resources/test_mutation_effect_summaries.tsv";
+
+    private String gene;
+    private String variant;
+    private String mutationEffect;
+    private String description;
+
+    public MutationEffectParameterizedTest(String gene, String variant, String mutationEffect, String description) {
+        this.gene = gene;
+        this.variant = variant;
+        this.mutationEffect = mutationEffect;
+        this.description = description;
+    }
+
+    @Test
+    public void testSummary() throws Exception {
+        Query query = new Query();
+        query.setHugoSymbol(gene);
+        query.setAlteration(variant);
+
+        IndicatorQueryResp resp = IndicatorUtils.processQuery(query, null, false, null);
+        String _query = gene + " " + variant;
+        String _mutationEffect = resp.getMutationEffect().getKnownEffect();
+        String _description = resp.getMutationEffect().getDescription();
+//        System.out.println("New: " + gene + "&&" + variant + "&&" + _mutationEffect + "&&" + _description);
```

Comment:
Still needed?

Reply:
Yup, this is used in all tests to populate all tests so we don't have to modify individually when there are a lot of valid tests to be modified.